pkg/common/prometheus/metrics.go

```diff
@@ -42,6 +42,24 @@ const (
 	// PrometheusExpandVolumeOpType represents the ExpandVolume operation.
 	PrometheusExpandVolumeOpType = "expand-volume"
 
+	// CSI operation fault types
+
+	// PrometheusFaultInvalidArgumentType represents the InvalidArgument fault
+	// that CSI operation may failed with
```

Comment:
Nits. Punctuation in comments.

Can you end your sentence with a "."? We could treat comments as a small article to help
others read code. They should follow the right grammar and syntax.

"may failed with" => "throws on failure"?